corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -53,17 +55,10 @@ def post(self, request, *args, **kwargs):
 
         # reset the session's expiry if there's some formplayer activity
         secure_session = session.get('secure_session')
-        if secure_session:
-            timeout = settings.SECURE_TIMEOUT
-            domain = data.get('domain')
-            if domain:
-                domain_obj = Domain.get_by_name(domain)
-                if domain_obj:
-                    timeout = domain_obj.secure_timeout or timeout
-        else:
-            timeout = settings.INACTIVITY_TIMEOUT
+        timeout = settings.SECURE_TIMEOUT if secure_session else settings.INACTIVITY_TIMEOUT
 
         session.set_expiry(timeout * 60)
+        session['last_request'] = json_format_datetime(datetime.datetime.utcnow())
```

Comment:
I missed doing this in the original logout workflow PR, https://github.com/dimagi/commcare-hq/pull/27630/ and, as a result, ended up thinking that web apps requests weren't correctly extending the session. This led to adding a client-side activity tracker in https://github.com/dimagi/commcare-hq/pull/27680